Dockerfile

```diff
@@ -75,6 +75,8 @@ COPY presto-kudu /build/presto-kudu
 COPY presto-main /build/presto-main
```

Comment:
We also need to emulate these changes that were required here, to the other (non-rhel) Dockerfiles. We should probably work on consolidating those Dockerfile too at some point.